mobile/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import { Button, Pressable, Text, View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+type Properties = {
+  isVisible: boolean;
+  onClose: () => void;
+  onDelete?: () => void;
+  type?: 'Note';
+  children?: React.ReactNode;
+};
+
+const Modal: React.FC<Properties> = ({
+  isVisible,
+  onClose,
+  onDelete,
+  type,
+}) => {
+  return (
+    <RNModal visible={isVisible} transparent animationType="none">
+      <Pressable onPress={onClose} style={styles.background}>
+        <View style={styles.modalContainer}>
+          <Pressable>
+            <View style={styles.modalView}>
+              {onDelete ? (
```

Comment:
```suggestion
              {Boolean(onDelete) && (
```

Reply:
@v0ldemar01 i need conditional rendering of 2 different components here